iocage_lib/ioc_plugin.py

```diff
@@ -846,6 +852,17 @@ def fetch_plugin_index(
                 if icon:
                     list_header += ["ICON"]
 
+                plugin_list = [
```

Comment:
I am a little out of context, so sorry if I am saying nonsense, but you seem to be removing cells with empty values from table rows, shifting rightmost cells to the left, breaking table structure (cols won't match neither with headers nor with cols in other rows), is this what you are doing?

Reply:
@themylogin nope. `name`, `description`, `pkg` are always present. `icon` is controlled by variable, so if `icon` is false, the dictionary won't have it, and it won't be included in the list for the table. Table structure would remain intact that way with/without icon supplied.

Reply:
Can it be more clear then? What protects us from empty `description`?

Reply:
@themylogin updated